### PR TITLE
Meenu fix responsive scrollbar

### PR DESCRIPTION
**Pull request by meenakshi-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/features/components/templates/navigation/template/mobile-menu/mobile-menu.module.scss [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
-    inline-size: $menu_size;
+    inline-size: 100%;
     block-size: calc(100vh - 7rem);
     background-color: $color-white;
     opacity: 1;
-    inset-inline-start: -$menu_size;
+    inset-inline-start: -100vh;
     overflow: scroll;
-    transition: inset-inline-start 0.5s ease 0ms;
+    transition:  0.5s ease 0ms;
```
Comment (hasan-deriv) [inline src/features/components/templates/navigation/template/mobile-menu/mobile-menu.module.scss:12]: Why did you remove `inset-inline-start` ?

**Review by hasan-deriv — APPROVED** [review]

**Review by mayuran-deriv — APPROVED** [review]

File: src/features/components/templates/navigation/template/mobile-menu/mobile-menu.module.scss (hunk 1 above)
Comment (meenakshi-deriv) [reply src/features/components/templates/navigation/template/mobile-menu/mobile-menu.module.scss:12]: Previously it was given a width of 30.8rem, but as per new design we are going to have full width. So it's not needed.

File: src/features/components/templates/navigation/template/index.tsx [hunk 2]
```diff
@@ -44,6 +44,7 @@ const NavTemplate = ({
 
     const onMenuToggleClick = () => {
         setIsMenuOpen((prevState) => !prevState)
+        document.body.style.overflow = is_mobile_or_tablet && is_menu_open ? 'scroll' : 'hidden'
```
Comment (Nuzhy-Deriv) [inline src/features/components/templates/navigation/template/index.tsx:47]:
```suggestion
        const lang_style = document.getElementById('lang_scroll_style')
        if (lang_style && is_menu_open) lang_style.remove()
        document.body.style.overflow = is_mobile_or_tablet && is_menu_open ? 'scroll' : 'hidden'
```

**Review by Nuzhy-Deriv — APPROVED** [review]

**Review by mohammad-hashemi-deriv — APPROVED** [review]

**Review by habib-deriv — APPROVED** [review]
LGTM
